python/triton/language/math.py

```diff
@@ -173,9 +173,9 @@ def rsqrt(x, _builder=None):
     return core.tensor(_builder.create_rsqrt(x.handle), x.type)
 
 
+@core._tensor_member_fn
 @core.builtin
 @_add_math_1arg_docstr("absolute value")
-@core._tensor_member_fn
```

Comment:
Was seeing a test failure:
```
AttributeError("\'NoneType\' object has no attribute \'create_fabs\'")
```
It seems that `tensor.abs()` wasn't being patched correctly and moving the `builtin` decorator before fixes it.